sentry/src/main/scala/com/avast/sst/sentry/SentryModule.scala

```diff
@@ -26,16 +26,20 @@ object SentryModule {
   }
 
   /** Makes [[io.sentry.SentryClient]] initialized with the given config and overrides the `release` property
-    * with `Implementation-Version` from the `MANIFEST.MF` file inside the same JAR (package) as the `Main` class.
+    * with `Implementation-Title`@`Implementation-Version` from the `MANIFEST.MF` file inside the same JAR (package) as the `Main` class.
+    *
+    * This format is recommended by Sentry ([[https://docs.sentry.io/workflow/releases]])
+    * because releases are global and must be differentiated.
     */
   def makeWithReleaseFromPackage[F[_]: Sync, Main: ClassTag](config: SentryConfig): Resource[F, SentryClient] = {
     for {
       customizedConfig <- Resource.liftF {
         Sync[F].delay {
           for {
             pkg <- Option(implicitly[ClassTag[Main]].runtimeClass.getPackage)
-            v <- Option(pkg.getImplementationVersion)
-          } yield config.copy(release = Some(v))
+            title <- Option(pkg.getImplementationTitle)
+            version <- Option(pkg.getImplementationVersion)
+          } yield config.copy(release = Some(s"$title@$version"))
```

Comment:
If I understand this correctly, `release` will not be set if `ImplementationTitle` or `ImplementationVersion` is missing, right?

I have never set `Implementation-Title` - maybe it's my fault. I would consider to fallback to `pkg.getName`.